packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -109,14 +109,16 @@ export const useScaffoldEventHistory = <
 
   const isContractAddressAndClientReady = Boolean(deployedContractData?.address) && Boolean(publicClient);
 
+  const fromBlockValue = fromBlock !== undefined ? fromBlock : BigInt(deployedContractData?.deployedOnBlock || 0);
```

Comment:
I was thinking why not export `deployedOnBlock` by default as `bigInt` from the `genearteTsAbi.ts` script. 

But then seems like ponder accepts block as number only instead of bigInt https://ponder.sh/docs/config/contracts#example 

So it's fine we just export it default as number 🙌